sessionctx/variable/session.go

```diff
@@ -872,6 +872,12 @@ type SessionVars struct {
 	// not limit and spill will never happen
 	TiFlashMaxBytesBeforeExternalSort int64
 
+	// TiFlashEnablePipelineMode means if we should enable tiflash pipeline model to execute query or not.
```

Comment:
Will `TiFlashExecutionModel` be better?

Reply:
After discussion, it is now changed to `bool tidb_enable_tiflash_pipeline_model{true, false}`